packages/authentication/src/inactivy-guard.tsx

```
@@ -0,0 +1,42 @@
+  const onLogout = useCallback(async () => {
+    logout();
+    router.push('/network');
+  }, [router]);
+
+  const onReset = useCallback(() => {
+    if (timer.current) clearTimeout(timer.current);
+    timer.current = setTimeout(onLogout, maxInactivity);
+  }, [onLogout]);
+
+  useEffect(() => {
+    if (typeof window === undefined) return;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)